NOpenType/Transformation.cs

```diff
@@ -31,5 +31,13 @@ public Point<short> Transform(Point<short> p)
             var y = (((long)_m10 * px + (long)_m11 * py) >> 14) + _dy;
             return new Point<short>((short)x, (short)y);
         }
+        public Point<short> TransformNormal(Point<short> p)
+        {
+            var px = (long)p.X;
+            var py = (long)p.Y;
+            var x = (((long)_m00 * px + (long)_m10 * py) >> 14) + _dx;
+            var y = (((long)_m01 * px + (long)_m11 * py) >> 14) + _dy;
+            return new Point<short>((short)x, (short)y);
+        }
```

Comment:
Haha,
> Where in the truetype spec does it says you need normal transformation? It's probably just that they store R matrix in column major order

you are right, I don't think that way.

That fix (TransformNormal()),
I bring it from here ...
https://github.com/MikePopoloski/SharpFont/blob/master/SharpFont/Internal/Geometry.cs#L88